apis/core/v1alpha1/groupversion_info.go

```diff
@@ -25,11 +25,11 @@ import (
 )
 
 var (
-	// GroupVersion is group version used to register these objects
-	GroupVersion = schema.GroupVersion{Group: "core.openfunction.io", Version: "v1alpha1"}
+	// SchemeGroupVersion is group version used to register these objects
```

Comment:
I think we can keep GroupVersion as it is and add SchemeGroupVersion for client as keda does https://github.com/kedacore/keda/blob/main/apis/keda/v1alpha1/groupversion_info.go

Reply:
This is a very good suggestion, i will modify it like this.